mmaction/datasets/pipelines/loading.py

```diff
@@ -1275,7 +1276,14 @@ def __call__(self, results):
 
         offset = results.get('offset', 0)
 
-        for frame_idx in results['frame_inds']:
+        cache = {}
+        for i, frame_idx in enumerate(results['frame_inds']):
+            if frame_idx in cache:
+                imgs.append(cp.deepcopy(imgs[cache[frame_idx]]))
```

Comment:
Write a comment to explain that this line is used for avoiding loading duplicated frames